src/main/java/org/jabref/logic/importer/fileformat/RisImporter.java

```diff
@@ -300,4 +314,23 @@ private void addDoi(Map<String, String> hm, String val) {
             hm.put(FieldName.DOI, doi);
         }
     }
+
+    private boolean isDateTag(String s) {
+        for (String dateTag : dateTags) {
```

Comment:
This way of searching for a value is inefficient, it takes O(n) .e.g linear time. Although this is not that relevant here, I would nonetheless  use a stream, e.g directly white the method as:
 ` return dateTags.stream().anyMatch(date->date.equals(searchDatea)`

And please no single chars for parameters-

Reply:
Used indexOf to find out if a tag is a date